pyface/qt/QtWebKit.py

```diff
@@ -28,6 +28,38 @@
         from PyQt5.QtWebKit import *
         from PyQt5.QtWebKitWidgets import *
 
+elif qt_api == "pyqt6":
+    from PyQt5.QtWidgets import *
+
+    try:
+        from PyQt6.QtWebEngine import *
```

Comment:
`PyQt6.QtWebEngine` doesn't seem to exist (but `PyQt6.QtWebEngineCore` and `PyQt6.WebEngineWidgets` do).